src/store/slices/userSlice.ts

```diff
@@ -0,0 +1,28 @@
+import {createSlice} from "@reduxjs/toolkit";
+
+const initialState={
+    email:null,
+    token:null,
+    id:null
+};
+
+const userSlice=createSlice({
+    name:'user',
+    initialState,
+    reducers:{
+        setUser(state,action){
+            state.email=action.payload.email;
+            state.token=action.payload.token;
+            state.id=action.payload.id;
+        },
+        removeUser(state){
+            state.email=null;
+            state.token=null;
+            state.id=null;
+        },
+    },
+})
+
+export const {setUser,removeUser}=userSlice.actions
+
+export default userSlice.reducer;
```

Comment:
why ? and where are we using it ?

Reply:
When registering, not only send it to firebase, but also to the store, then on the Home page, when you click on the button, you clear the store and exit the Home page